scipy/_lib/_array_api.py

```diff
@@ -234,6 +234,17 @@ def xp_copy(x: Array, *, xp: ModuleType | None = None) -> Array:
     return _asarray(x, copy=True, xp=xp)
 
 
+def xp_sinc(x: Array, *, xp=None):
+    """`xp.sinc` replacement. Looking at you, array_api_strict.
```

Comment:
The only comment I can find on the array API side is https://github.com/data-apis/array-api/issues/8#issuecomment-656063277.

Reply:
You can replace this now with

```python
from scipy._lib import array_api_extra as xpx
...
xpx.sinc(...)
```

Reply:
OK thanks, done.

Reply:
this helper can be removed now!